### ISSUE TITLE
Duplicate multisite entries with supertables

### ISSUE THREAD

**Issue by pieter-janDB** [craftcms/cms#3716]
### Description

Hi, I have a craft 3 multisite setup with 2 languages. I use a supertable to input an image/label/lightswitch.
My problem occured when I want to duplicate an entry from the control panel or inside of an entry by choosing save as a new entry.

Scenario 1:
If I have "Manage blocks on a per-site basis" checked it will make a copy of the entry but leave my second language empty (other fields with the right locale data are copied, only the supertable is empty)

Scenario 2:
If I do not have "Manage blocks on a per-site basis" checked it gives me an error "Could not duplicate elements due to validation errors.".

- if I go into the entry with devmode enables I get redirected to a 500 error:

```
Next yii\web\ServerErrorHttpException: An error occurred when duplicating the entry. in /............./vendor/craftcms/cms/src/controllers/EntriesController.php:521
Stack trace:
#0 [internal function]: craft\controllers\EntriesController->actionSaveEntry()
#1 /......../vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array(Array, Array)
#2 /......../vendor/yiisoft/yii2/base/Controller.php(157): yii\base\InlineAction->runWithParams(Array)
#3 /......../vendor/craftcms/cms/src/web/Controller.php(104): yii\base\Controller->runAction('save-entry', Array)
#4 /......../vendor/yiisoft/yii2/base/Module.php(528): craft\web\Controller->runAction('save-entry', Array)
#5 /......../vendor/craftcms/cms/src/web/Application.php(282): yii\base\Module->runAction('entries/save-en...', Array)
#6 /......../vendor/craftcms/cms/src/web/Application.php(542): craft\web\Application->runAction('entries/save-en...', Array)
#7 /......../vendor/craftcms/cms/src/web/Application.php(266): craft\web\Application->_processActionRequest(Object(craft\web\Request))
#8 /......../vendor/yiisoft/yii2/base/Application.php(386): craft\web\Application->handleRequest(Object(craft\web\Request))
#9 /......../public_html/index.php(21): yii\base\Application->run()
#10 /........./.composer/vendor/laravel/valet/server.php(151): require('/Users/pieterja...')
#11 {main}
Copy Stacktrace Search Stackoverflow Search Google Exception
HTTP 500 – Internal Server Error – yii\web\ServerErrorHttpException
An error occurred when duplicating the entry.
↵
Caused by: Exception – yii\base\Exception
Element 468 doesn’t exist in the site 2
in /......../vendor/craftcms/cms/src/services/Elements.php at line 601
```

--> What I really need is to duplicate an array using the second scenario without "Manage blocks on a per-site basis" checked and only the label field translateable (translation method == translate for each language)

Note: Not sure if this is resolvable by craft or super table itself..

### Steps to reproduce

1. Setup a multisite with an entry that exists on 2 languages and has a supertable field.
2. duplicate that field.

### Additional info

- Craft version: 3.0.36
- PHP version: 7.1.25
- Database driver & version: mysqlnd 5.0.12-dev
- Plugin Super table  version=dev-craft-3

**Comment by TomDeSmet** [comment]
I have the same issue just today. Any response on this please?

**Comment by brandonkelly** [comment]
Make sure everything’s up to date. If so and you’re still seeing this, send a database backup and your `composer.json` + `composer.lock` files over to support@craftcms.com and we can look into it.

**Comment by mikegraham00** [comment]
I'm also seeing this. Craft and all plugins are up to date. Here is the error that was logged. Do you need the full stack trace to diagnose?

`2019-06-18 10:49:51 [-][1][6803ce29326f5d161d6ab820b759a0ef][error][yii\web\HttpException:500] PDOException: SQLSTATE[23000]: Integrity constraint violation: 1062 Duplicate entry '4223-1' for key 'craft_content_elementId_siteId_unq_idx' in /home/epicheli/epichelicopters/v2/vendor/yiisoft/yii2/db/Command.php:1290`

**Comment by brandonkelly** [comment]
At this point a couple people have sent in their files and we’ve found that the issue is resolved in Craft 3.2 Beta, thanks to some refactoring of `duplicateElements()`. So going to close this out. Craft 3.2 Stable will be released in the coming weeks.
